### PR TITLE
TypeScript Namespaces Release

### PR DESCRIPTION
**Pull request by zengin**
### Changes

- `CallRecords` namespace is exported as nested namespace.
- `callRecords` property is added to `CloudCommunications`

### Work Items
[AB#5029](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/5029)

### CODE REVIEW

File: microsoft-graph.d.ts [hunk 1]
```diff
@@ -13872,3 +13873,459 @@ export interface TimeRange {
     // End time for the time range.
     endTime?: NullableOption<string>;
 }
+
+
+export namespace CallRecords {
```
Comment (MIchaelMainer) [inline microsoft-graph.d.ts:13878]: Inconsistent capitalization between the `microsoftgraph` namespace and `CallRecords` namespace.

Comment (zengin) [reply microsoft-graph.d.ts:13878]: discussed offline, we agreed on keeping this PascalCase.

**Comment by zengin** [comment]
Converting this to draft to get linting changes in as well.

**Comment by zengin** [comment]
Republishing this PR, latest commit https://github.com/microsoftgraph/msgraph-typescript-typings/pull/184/commits/a56ef88ad34a2af0a5eec76c548ff6e07dab818e is the one with the fixes from https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/279

**Comment by zengin** [comment]
@baywet, please review the latest commit, which makes the linter happy.

**Pull request merged**